libs/configuration-variables-resolver/.gitignore

```diff
@@ -2,7 +2,8 @@ vendor/
 .idea/
 composer.lock
 /.php_cs.cache
-/.env
+/.env.*.local
+/.env.local
```

Comment:
Proč? To je zbytečný ne? Máš jeden env file pro celý monorepo. Jako jo, není to v readme, ale stejně to tak je.

Reply:
No nevím jak to je, ale když pouštíš testy lokálně, tak chceš ty envy mít spíš jen pro tu konkrétní lib ne?

Reply:
No ale tady je vetsina envu zatim spolecnych, takze mam normalne env file v rootu a pak volam

`docker-compose --profile dev run --rm dev-input-mapping coChci`

Reply:
a v phpstormu to máš taky přes docker compose?

Reply:
pres ten to nepoustim :)